server/rest/src/main/java/org/infinispan/rest/resources/ContainerResource.java

```diff
@@ -257,7 +260,26 @@ public Invocations getInvocations() {
          cachesHealth.put(ch.getCacheName(), ch.getStatus());
       }
 
+      LocalTopologyManager localTopologyManager = null;
+      try {
+         localTopologyManager = SecurityActions.getGlobalComponentRegistry(cacheManager).getLocalTopologyManager();
+      } catch (Exception e) {
+         // Unable to get the component. Just ignore.
+      }
+
+      Boolean clusterRebalancingEnabled = null;
+
+      if (localTopologyManager != null) {
+         try {
+            clusterRebalancingEnabled = localTopologyManager.isRebalancingEnabled();
+         } catch (Exception ex) {
+            // Something went wrong. Just ignore.
+         }
+      }
```

Comment:
We can just make this a single try/catch:

```java
      LocalTopologyManager localTopologyManager = null;
      Boolean clusterRebalancingEnabled = null;
      try {
         localTopologyManager = SecurityActions.getGlobalComponentRegistry(cacheManager).getLocalTopologyManager();
         clusterRebalancingEnabled = localTopologyManager.isRebalancingEnabled();
      } catch (Exception e) {
         // Unable to get the component. Just ignore.
      }
```

Reply:
ok